packages/xstate-react/src/useSpawn.ts

```diff
@@ -16,5 +17,10 @@ export function useSpawn<TState, TEvent extends EventObject>(
     return spawnBehavior(behavior);
   });
 
+  useEffect(() => {
+    actorRef.start!();
+    return () => actorRef!.stop!();
+  });
```

Comment:
Top-level actorRefs are required to have `start` method because we need to start them in a "deferred" manner. We can't start them when rendering in the React world - this has to be done in an effect